sway/commands.c

```diff
@@ -166,7 +166,7 @@ static bool cmd_exit(struct sway_config *config, int argc, char **argv) {
 	}
 	// Close all views
 	container_map(&root_container, kill_views, NULL);
-	exit(0);
+	wlc_terminate();
```

Comment:
The other places are in sway_abort (where it's okay, i guess) and when forking (perfectly fine there).